README.md

```diff
@@ -54,6 +54,23 @@ $classInfo = ReflectionClass::createFromName('Foo\Bar\MyClass');
  * `ReflectionType::createFromType()` now requires `string` for `$type` parameter
  * `ReflectionFunctionAbstract::setReturnType()` now requires `string` for `$newReturnType` parameter
  * `SourceStubber::__invoke()` now requires `\ReflectionClass` for `$classReflection` parameter
+ * `Roave\BetterReflection\Reflector\FunctionReflector::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Ast\Locator::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\AbstractSourceLocator::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\AutoloadSourceLocator::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\ClosureSourceLocator::__construct` has additional mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\ComposerSourceLocator::__construct` has additional mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\DirectoriesSourceLocator::__construct` has additional mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\EvaledCodeSourceLocator::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\FileIteratorSourceLocator::__construct` has additional mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\PhpInternalSourceLocator::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\SingleFileSourceLocator::__construct` has additional mandatory constructor arguments
+ * `Roave\BetterReflection\SourceLcoator\Type\StringSourceLocator::__construct` has additional mandatory constructor arguments
+ * `Roave\BetterReflection\Util\FindReflectionOnLine::__construct` now has mandatory constructor arguments
+ * `Roave\BetterReflection\Reflector\ClassReflector::buildDefaultReflector()` was dropped, please use the newly
+   introduced `Roave\BetterReflection\BetterReflection#classReflector()` instead
```

Comment:
Would not be better to use `::` instead of `#` ?

Reply:
`#` for instance methods, `::` for static methods